electron/github-electron.d.ts

```diff
@@ -2655,6 +2655,28 @@ declare namespace Electron {
 		 * Returns a boolean whether the image is a template image.
 		 */
 		isTemplateImage(): boolean;
+		/**
+		 * @param rect The area of the image to crop
+		 * @returns The cropped image.
+		 */
+		crop(rect: Rectangle): NativeImage;
+		/**
+		 * @returns The resized image.
+		 * If only the height or the width are specified then the current aspect ratio will be preserved in the resized image.
+		 */
+		resize(options: {
```

Comment:
https://github.com/electron/electron/blob/master/docs/api/native-image.md#imageresizeoptions

According to the documentation it's not optional.